jira-macro/jira-macro-default/src/main/java/org/xwiki/contrib/jira/macro/internal/displayer/AbstractJIRADisplayer.java

```diff
@@ -124,6 +140,26 @@ private void normalizeField(JIRAField field)
         }
     }
 
+    /**
+     * Provide a block to show to the user when the displayer is wrongly used inline.
+     *
+     * @param displayerName the name of the displayer.
+     */
+    protected List<Block> getInlineNotSupportedError(String displayerName)
+    {
+        List<Block> errorBlocks = List.of();
+        try {
+            errorBlocks =
+                parser.parse(
```

Comment:
You might want to use the `ErrorBlockGenerator`. Also, it would probably make sense to translate this message (`ErrorBlockGenerator#generateErrorBlocks` supports a translation key).